### PR TITLE
New tests for pecuzal, uzal_cost; corrected normalization in uzal cost

### PR DESCRIPTION
**Pull request by hkraemer**
This is regarding #95, #96 and also #94. The correction in the computation of Uzal's cost function have been implemented and the according tests have been revised. Same holds for the brittle PECUZAL-tests. In the PECUZAL case I double-checked each test with the outcomes from my [Matlab](https://github.com/hkraemer/PECUZAL_Matlab) and [Python](https://github.com/hkraemer/PECUZAL_Python) implementations of the algorithm.  I also relaxed the test-constraints and outsourced the time series being used (we should continuing doing this for the other methods). Hopefully this works now. Cheers

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -12,6 +14,13 @@ for a in todownload
     download(repo*"/"*a, joinpath(tsfolder, a))
 end
 
+#New:
+repo = "https://raw.githubusercontent.com/JuliaDynamics/JuliaDynamics/master/timeseries"
+a = "test_time_series_lorenz_standard_N_10000_multivariate.csv"
+download(repo*"/"*a, joinpath(tsfolder, a))
+a = "test_time_series_roessler_N_10000_multivariate.csv"
+download(repo*"/"*a, joinpath(tsfolder, a))
+
```
Comment (Datseris) [inline test/runtests.jl:23]: Should be written in terms of a loop. Define `files = [all file names here]` and loop over `for a in files`

**Comment by hkraemer** [comment]
hmpf, I think I PR'ed from an old branch... hm. Might not be a problem hopefully.

**Comment by Datseris** [comment]
it's okay, no conflicts. tests pass, do you need to add anything more here or the PR should be merged?

**Comment by hkraemer** [comment]
no I think I'll do the loop thingy and that's it

**Comment by hkraemer** [comment]
I'll fix the other two test next week or so. Super busy atm :(

**Comment by Datseris** [comment]
No problem, take your time. Don't forget that we need to tag a new release for these changes to be propagated beyond development in a stable release. You should tell me when you feel a new version should come out and I'll tag it.

**Pull request merged**